README.md

```diff
@@ -20,6 +20,41 @@ To sign in to Chartio:
 - Open the [FT Chartio](https://chartio.com/financialtimes/) in your browser.
 - Press the "Sign in with Google" button and select your FT Google Account.
 
+### How to track extra click data using `data-trackable` attributes
+
+You can add extra information to o-tracking events by using the `data-trackable` and `data-trackable-context-*` custom attributes. 
+
+
+#### `data-trackable`
+
+`o-tracking` will walk up the DOM to get the tree of elements with `data-trackable` attributes. For example, clicks to the list-item button below will result in an event that has this path recorded within it - `header-menu country-selector china`:-
+
+```html
+<div data-trackable="header-menu">
+	<span>
+		<div data-trackable="country-selector">
+			<ul>
+				<li data-trackable="china" role="button">China</li>
+			</ul>
+		</div>
+	</span>
+</div>
+```
+
+#### `data-trackable-context-*`
+
+To add extra context to events you may add custom attributes in the form: `data-trackable-context-name="data"` where `name` is the name of the extra context and `data` is the extra data.
+
+For example, when the anchor element is clicked, it will add extra event fields `article_guid` and `article_source` to the data being tracked.
```

Comment:
```suggestion
For example, when the below anchor element is clicked, it will add extra event fields `article_guid` and `article_source` to the data being tracked.
```